examples/hello_world.rs

```diff
@@ -63,25 +67,26 @@ fn main() -> anyhow::Result<()> {
         // We use the default namespace "env".
         "env" => {
             // And call our function "say_hello".
-            "say_hello" => Function::new_native(&store, say_hello_world),
+            "say_hello" => Function::new_native(&mut context, say_hello_world),
         }
     };
 
     // We then use the `Module` and the import object to create an `Instance`.
     //
     // An `Instance` is a compiled WebAssembly module that has been set up
     // and is ready to execute.
-    let instance = Instance::new(&module, &import_object)?;
+    let instance = Instance::new(&mut context, &module, &import_object)?;
 
     // We get the `TypedFunction` with no parameters and no results from the instance.
     //
     // Recall that the Wasm module exported a function named "run", this is getting
     // that exported function from the `Instance`.
-    let run_func: TypedFunction<(), ()> = instance.exports.get_native_function("run")?;
+    let run_func: TypedFunction<(), ()> =
+        instance.exports.get_native_function(&mut context, "run")?;
```

Comment:
Can we mark the function `get_native_function` as deprecated, and points to use a new `get_typed_function` instead?